docs/wxm-devices/d1/connect-and-claim.mdx

```diff
@@ -101,6 +101,12 @@ While you are connected to the "WXM_WG1200" network, you will not have internet
 
 <hr/>
```

Comment:
Please remove this divider and we are good to go.

Reply:
ok Thanks